CoverageExt/CoverageExecution.cs

```diff
@@ -140,7 +163,7 @@ private void StartImpl(string solutionFolder, string platform, string dllFolder,
                     {
                         // Kill process.
                         process.Kill();
-                        throw new Exception("Creating code coverage timed out.");
+                        throw new Exception("Creating code coverage timed out (more than 15min).");
```

Comment:
Yeah I'm still not sure if this should be 15 minutes. This actually sais: 15 minutes without activity on the command line. Perhaps I should create a ticket to improve this. It's quite easy to monitor CPU process as well; if there's no activity there as well, we can just consider it 'waiting for nothing'... (e.g. readline) 
For now this is fine.